R/rlistings.R

```diff
@@ -13,22 +13,51 @@ setOldClass(c("MatrixPrintForm", "list"))
 #' @param cols character. Names of columns (including but not limited to key columns)
 #'   which should be displayed when the listing is rendered.
 #' @param key_cols character. Names of columns which should be treated as *key columns*
-#'   when rendering the listing.
+#'   when rendering the listing. Key columns allow you to group repeat occurrences.
 #' @param main_title character(1) or NULL. The main title for the listing, or
 #'   `NULL` (the default). Must be length 1 non-NULL.
 #' @param subtitles character or NULL. A vector of subtitle(s) for the
 #'   listing, or `NULL` (the default).
 #' @param main_footer character or NULL. A vector of main footer lines
 #'   for the listing, or `NULL` (the default).
 #' @param prov_footer character or NULL. A vector of provenance strings
-#'   for the listing, or `NULL` (the default).
-#'
+#'   for the listing, or `NULL` (the default). Each string element is placed on a new line.
 #' @return A `listing_df` object, sorted by the key columns.
 #'
+#' @details
+#' key_cols is useful when a patient experiences more than 1 adverse event, rather than
+#' listing out a separate row for each adverse event, they are able to be grouped
+#' together for each patient. These columns are always present in the listing even if
```

Comment:
even if? I would keep all of this a bit more general, as it is not always patients or AE we want to have as key_cols